src/components/info/info.model.ts

```diff
@@ -0,0 +1,24 @@
+import mongoose, { Schema } from 'mongoose'
+
+export interface InfoInterface {
+  iconsAdded: string[],
+  iconsDeleted: string[],
+  iconsUpdated: string[],
+  }
+
+// Main Schema:
+const InfoSchema = new Schema({
+  iconsAdded: { type: [String], required: true },
+  iconsDeleted: { type: [String], required: true },
+  iconsUpdated: { type: [String], required: true }
+}, {
+  timestamps: true,
+  collection: 'info'
+})
+// Indexes
+InfoSchema.index({ createdAt: -1 })
```

Comment:
Why are we creating indexes like this? We can directly do this from atlas for once. I saw some docs of mongoose and they say it affects performance in production (https://mongoosejs.com/docs/guide.html#indexes).

Reply:
I think the issue is that if we kept `autoIndex` set to true, it will try to re-create the indexes if they weren't found in the application startup, but this's most probably will not be the case, since the indexes will be always there, and the `ensureIndexes` calls will return false, so it will not be re-created. 
I've understood more from the answer in this question [Mongoose indexing in production code
](https://stackoverflow.com/questions/14342708/mongoose-indexing-in-production-code)

Reply:
Do you think we need to disable it? I'm thinking of that it would be nice to check if all indexes are existent, in case if we decided to add more indexes in the future using the code, but in the same time I don't think so this will usually happen.

Reply:
I think yes, checking indexes is a good thing but if there is even a little chance of it affecting the performance, then we shouldn't.

Reply:
If it tries to re-create indexes (which i guess it wont, if autoIndex is false), the analytics collection has a lot of documents, it will take a lot of time.

Reply:
As per my understand, the indexes will not be removed unless someone removed it manually using Atlas, so it should never re `re-created` in anytime in the future. (we will never delete them using atlas)

Reply:
If you're think that it would be an overhead in case if it was deleted by mistake and mongoose go through the re-creation process, it should be disabled. 
My argument is that the probability of this case is probably zero, I don't think so that we will ever attempt to remove the indexes from atlas, and keeping them in the code. 
(Except if the indexes can be deleted someway other than manually)

Reply:
You're right. Probably the indexes won't get deleted.